pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go

```diff
@@ -63,6 +64,21 @@ func (mutator *VMIsMutator) Mutate(ar *admissionv1.AdmissionReview) *admissionv1
 	if ar.Request.Operation == admissionv1.Create {
 		informers := webhooks.GetInformers()
 
+		hugepages := false
```

Comment:
It would be good to comment here and document that hugepage does not work with qemu session mode. Can it be pick up by kubevirt documents?